stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+apiVersion: v1
+kind: List
```

Comment:
I find this hard to review as a single list. Would you mind splitting into individual manifests?